freeze/urls.py

```diff
@@ -1,12 +1,17 @@
 # -*- coding: utf-8 -*-
 
-from django.conf.urls import url
-
+from django.urls import path
```

Comment:
Please add conditional import depending on django version for bacward-compatibility.